### DISCUSSION TITLE
AdGuard home and VLANs

### DISCUSSION THREAD

**Discussion by cgi2099** [AdguardTeam/AdGuardHome#7188] [Q&A]
Hello all, I have AdGuard Home installed on my OPNSense router. I am needing the ability to apply different block lists to different VLANs. For instance I have these Networks:

192.168.1.0/24
10.10.10.0/24
10.10.20.0/24

I need 192.168.1.0/24 & 10.10.10.0/24 to share the same blocklist and rules.

But I need 10.10.20.0/24 to have a different blocklist and rules.

Is this possible? I do not see any options in the GUI where this can be done.

Thank you,
Josh

**Comment by nickle90** [comment]
Simply create a client object under Settings -> Client Settings with the object of 10.10.20.1/24 under Identifier and put whichever settings you want there

**Reply by cgi2099** [reply]
Thank you so much, I don't see a place on mine to select a different blocklist than everything else. Do I need to change something to get these options?

**Reply by nickle90** [reply]
![image](https://github.com/user-attachments/assets/5031ade2-c377-489d-99c6-c123fc793b26)

![image](https://github.com/user-attachments/assets/2e69e8e2-a5b4-4bea-916c-05ce45596e9e)

**Reply by al13nus** [reply]
You have to do this for every client. That's not convenient, specially when new client get added, you have to update the list.

**Reply by cgi2099** [reply]
I was unable to figure this out or get it to work. I need for each client to use a completely different blocklist and do not see the ability to do this anywhere.

**Reply by Dynamic5912** [reply]
You can't do that
